tests/test_standard_input_json.py

```diff
@@ -1,23 +1,169 @@
 
 class TestStandardInputFormatLoading(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.dss_client = hca.dss.DSSClient()
+        cls.dss_client.host = 'https://hca-dss-4.ucsc-cgp-dev.org/v1'
         cls.dss_endpoint = os.getenv("TEST_DSS_ENDPOINT", "https://hca-dss-4.ucsc-cgp-dev.org/v1")
         cls.staging_bucket = os.getenv("DSS_S3_STAGING_BUCKET", "mbaumann-dss-staging")
-        cls.project_path = Path(__file__).parents[1]
+        cls.test_file = TEST_DATA_PATH / 'gen3_sample_input_standard_metadata.json'
+
+    def test_data_matches_schema(self):
+        """This is a sanity check to make sure that the test data matches the agreed upon schema"""
+        test_json = json.loads(self.test_file.read_text())
+        for bundle in test_json:
+            jsonschema.validate(bundle, standard_schema)
 
-    @unittest.skip("Support for the basic import format is not implemented.")
     def test_basic_input_format_loading_from_cli(self):
-        # TODO Implement this test
-        # json_input_file = f"{self.project_path}/tests/test_data/standard_sample_input.json"
-        pass
+        self._test_gen3_loading_from_cli(self.test_file)
+
+    @staticmethod
+    @contextmanager
+    def _tmp_json_file(json_input_file, guid, file_guid, file_version):
+        """Yields a temporary test file with identifying information changed"""
+
+        def change_info(in_json, guid, file_guid, file_version):
+            only_bundle = in_json[0]
+            data_bundle = only_bundle['data_bundle']
+            data_objects = only_bundle['data_objects']
+
+            data_bundle['id'] = guid
+
+            # change all references to the file guid
+            object_key = data_bundle['data_object_ids'][0]
+            # FIXME file_guid doesn't have url prefix!
+            data_bundle['data_object_ids'][0] = file_guid
+            data_objects[file_guid] = data_objects[object_key]
+            del data_objects[object_key]
+            data_objects[file_guid]['id'] = file_guid
+
+            # update versions for all files
+            for file_ in data_objects.values():
+                file_['updated'] = file_version
+
+            return [only_bundle]
+
+        with open(json_input_file, 'r') as jsonFile:
+            json_contents = json.load(jsonFile)
+        fixed_json = change_info(json_contents, guid, file_guid, file_version)
+        with tempfile.NamedTemporaryFile() as jsonFile:
+            with open(jsonFile.name, 'w') as fh:
+                json.dump(fixed_json, fh)
+            yield jsonFile.name
+
+    def _load_file(self, tmp_json):
+        """run the load script and clean up after ourselves"""
+        # upload the data bundle to the DSS
+        args = ['--no-dry-run',
+                '--dss-endpoint',
+                f'{self.dss_endpoint}',
+                '--staging-bucket',
+                f'{self.staging_bucket}',
+                '--log',
+                'DEBUG',
```

Comment:
In general, tests should run at the default log level. An alternate log level should not be specified as part of the standard test input parameters.

Reply:
I set it like this because originally I was trying to debug Travis and needed the log. I found it very helpful as well when trying to debug failing tests with Toil. I think having the full log for unit tests can be valuable for these reasons. Can you convince me why it should be default?